dpdk-dev-libibverbs: suppress gcc7 warnings

We know that format-truncation in init.c is okay.

Upstream-Status: Pending

Signed-off-by: Joe Slater <joe.slater@windriver.com>


--- a/src/init.c
+++ b/src/init.c
@@ -31,6 +31,11 @@
  * SOFTWARE.
  */
 
+/* For this file we know we can ignore this warning. */
+#if __GNUC__ > 6
+#pragma GCC diagnostic ignored "-Wformat-truncation"
+#endif
+
 #if HAVE_CONFIG_H
 #  include <config.h>
 #endif /* HAVE_CONFIG_H */
